setup.py

```diff
@@ -16,12 +16,13 @@
     author_email='parsl@googlegroups.com',
     license='Apache 2.0',
     download_url='https://github.com/Parsl/parsl/archive/{}.tar.gz'.format(VERSION),
-    package_data={'': ['LICENSE']},
+    include_package_data=True,
     packages=find_packages(),
     install_requires=install_requires,
     scripts = ['parsl/executors/high_throughput/process_worker_pool.py',
                'parsl/executors/extreme_scale/mpi_worker_pool.py'],
     extras_require = {
+        'parsl-visualize': ['dash', 'dash-html-components', 'dash-core-components', 'pandas', 'sqlite3'],
```

Comment:
@garri1105 Remove sqlite3, it is part of the standard library now and will return an error if you try to install it.

Reply:
Done.